pkgs/aws/amazon-ec2-spot-interrupter/registry.yaml

```diff
@@ -0,0 +1,20 @@
+packages:
+  - type: github_release
+    repo_owner: aws
+    repo_name: amazon-ec2-spot-interrupter
+    asset: ec2-spot-interrupter_{{trimV .Version}}_{{.OS}}_{{.Arch}}.tar.gz
+    description: The ec2-spot-interrupter is a simple CLI tool that triggers Amazon EC2 Spot Interruption Notifications and Rebalance Recommendations
+    replacements:
+      darwin: Darwin
+      linux: Linux
+    supported_envs:
+      - linux
+      - darwin
+    overrides:
+      - goos: linux
+        goarch: arm64
+        replacements:
+          arm64: armv6
+    files:
+      - name: ec2-spot-interrupter
+        src: ec2-spot-interrupter
```

Comment:
```suggestion
```

You can omit `src` when src is equivalent to `name`.

Reply:
I didn't know! Thank you.